### ISSUE TITLE
Metabase Initialization FAILED

### ISSUE THREAD

**Issue by mrwalji** [metabase/metabase#2672]
If this is a bug report, please fill in the blanks:

*  I am using the FireFox browser.
*  My computer's OS is Centos 7
*  I'm running Metabase Locally. [locally from the JAR / locally with the Mac App / on AWS / on Herkou / etc.]
*  My database is MYSQL
*  My Metabase version is LATEST - YESTERDAY.

**Comment by mrwalji** [comment]
Ran fine under centos, set up with MYSQL and worked perfect, was setting up an email system and got error PermGen, exit application, restart and run again

turns out it is trying to recreate the tables in the mysql DB and failing

copy of error

```
ERROR metabase.core :: Metabase Initialization FAILED:  liquibase.exception.MigrationFailedException: Migration failed for change set migrations/001_initial_schema.json::1::agilliland:
     Reason: liquibase.exception.DatabaseException: Table 'core_organization' already exists [Failed SQL: CREATE TABLE metabase.core_organization (id INT AUTO_INCREMENT NOT NULL, slug VARCHAR(254) NOT NULL, name VARCHAR(254) NOT NULL, description TEXT NULL, logo_url VARCHAR(254) NULL, inherits BIT(1) NOT NULL, CONSTRAINT PK_CORE_ORGANIZATION PRIMARY KEY (id), UNIQUE (slug))]
05-26 11:24:39 INFO metabase.core :: Metabase Shutting Down ...
05-26 11:24:39 INFO metabase.core :: Metabase Shutdown COMPLETE
```

Any Idea

Thanks

**Comment by agilliland** [comment]
The only reason the migrations would try to run again is if they think they haven't already been executed, so something is causing that to happen.  The executed migration history is in the `DATABASECHANGELOG` table in your db.

When you look at your database, what's the list of tables?  and whats in the `DATABASECHANGELOG` table?

**Comment by camsaul** [comment]
@mrwalji are you still seeing this issue?

**Comment by camsaul** [comment]
The root cause of this issue is fixed by #3295; once Metabase 0.20 ships in the next few weeks you'll be able to fix your database using the new `migrate force` command:

```bash
java -jar metabase.jar migrate force
```
